README.md

```diff
@@ -1,68 +1,211 @@
----
-tags: readme
-language: ruby
-resources: 0
-track: web development
-topic: ruby
-unit: hashes
-lesson: overview, creation, manipulation, retrieval
----
-
 # Hashes
 
+## Objectives
+
+We're going to introduce a new data structure, one that is more complicated than arrays. Hashes will occupy us for the next several lessons and there's a lot to cover. This lesson, which includes code challenges, covers the following:
+
+1. Introduction: what are hashes and what do we need them for?
+2. Creating
+3. Retrieval
+4. Adding Data to Hashes
+
+In the upcoming lessons, we'll go over how to iterate over and manipulate hashes and how to build more complex, or nested, hashes. 
+
 ## Introduction
-* "Hashes are a lot like arrays: they have a bunch of slots that can point to various objects. However, in an array, the slots are lined up in a row, and each one is numbered (starting from zero). In a hash, the slots aren’t in a row (they are just sort of jumbled together) and you can use any object to refer to a slot, not just a number." - From [Chris Pine's Learn to Program, page 108](http://books.flatironschool.com/books/43?page=108)
+> Hashes are a lot like arrays: they have a bunch of slots that can point to various objects. However, in an array, the slots are lined up in a row, and each one is numbered (starting from zero). In a hash, the slots aren’t in a row (they are just sort of jumbled together) and you can use any object to refer to a slot, not just a number. 
+> 
+> –– [Chris Pine's Learn to Program, page 108](http://books.flatironschool.com/books/43?page=108)
+
+
+Up until this point, we've stored our data in list-form using arrays. Let's take a look at an example where an array would not be sufficient to meet our storage needs.
+
+### Example: Customs Inspector
+
+![Herman Melville](https://upload.wikimedia.org/wikipedia/commons/thumb/f/f1/Herman_Melville.jpg/160px-Herman_Melville.jpg)
```

Comment:
DL this and add it to a our S3. Don't want to steal bandwidth from wikimedia and want to protect against it going away